File: cli-go/zally/violation.go
package main

// Violation keeps information about Zally violations
type Violation struct {
	Title         string   `json:"title"`
	Decription    string   `json:"description"`
	ViolationType string   `json:"violation_type"`
	RuleLink      string   `json:"rule_link"`
	Paths         []string `json:"paths"`
}

File: server/src/main/java/de/zalando/zally/rule/Violation.kt
package de.zalando.zally.rule

import de.zalando.zally.dto.ViolationType

data class Violation(

    val rule: Rule,
    val title: String,
    val description: String,
    val violationType: ViolationType,
    val ruleLink: String,
    val paths: List<String>
)
